c10/core/ScalarType.h

```diff
@@ -467,6 +478,7 @@ inline bool isSignedType(ScalarType t) {
       CASE_ISSIGNED(Float8_e5m2fnuz);
       CASE_ISSIGNED(Float8_e4m3fn);
       CASE_ISSIGNED(Float8_e4m3fnuz);
+      CASE_ISSIGNED(Float8_e8m0fnu);
```

Comment:
I'm not sure about this.
If we expect to have numeric_limits for everything, let's just have a big macro for the whole thing.
If we're hardcoding it, let's just use the macro and a "true"/"false" below?